### ISSUE TITLE
Keyboard navigation for styles grids

### ISSUE THREAD

**Issue by mmotyczynska** [ckeditor/ckeditor5#12250]
## Provide a description of the task

There should be a proper navigation with keyboard for styles dropdown.
There are two grids - switching between them should be possible with TAB key.
In every grid there should be o proper navigation among items with arrow keys (right/left/up/down).

## 📃 Other details

We should use a new helper for grid navigation introduced in https://github.com/ckeditor/ckeditor5/pull/12202.

The dropdown we should work on:
![image](https://user-images.githubusercontent.com/95343030/183861265-72bb1352-84eb-40a9-874d-8903d2c55ae3.png)

* Browser: …
* OS: …
* CKEditor version: …
* Installed CKEditor plugins: …

**Comment by mlewand** [comment]
ckeditor/ckeditor5#12257 might be an inspiration.
